src/impl/FormulaCell.ts

```diff
@@ -10,10 +10,10 @@ export class FormulaCell<T> implements Cell<T>, Recalculable {
     private val: T
     private destroyed: boolean = false
     private readonly fn: Function
-    private readonly sources: (Cell<T> | T)[]
+    private readonly sources: Value<T>[]
```

Comment:
Actually, there was a mistake in types. `T` is the type of value that the formula stores. Sources at the other hand should not be `Value<T>`, they actually should be `unknown[]`